src/current/_includes/releases/v24.1/v24.1.1.md

```diff
@@ -0,0 +1,103 @@
+- CockroachDB now automatically repairs tables that see the error `invalid inbound foreign key ... origin table ID should be" or "invalid outbound foreign key ... reference table ID should be`. [#123668][#123668]
+- Fixed a bug where a failed [`RESTORE`]({% link {{ page.version.version }}/restore.md %}) could leave the system in a state where re-attempting the restore was not possible without manual intervention. [#123205][#123205]
+- Fixed a bug that could cause calling a routine to return an unexpected `function ... does not exist` error. The bug is triggered when the routine is called twice using the exact same SQL query, and either: (a) the routine has polymorphic arguments, or: (b) in between the two calls, the routine is replaced by a routine with the same name and different parameters. This bug has existed since alpha versions of v23.1. [#123516][#123516]
+- Fixed a rare bug where a lease transfer could lead to a `side-transport update saw closed timestamp regression` panic. The bug could occur when a node was overloaded and [failing to heartbeat its node liveness record]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#node-liveness-issues). [#123533][#123533]
+- Fixed a crash introduced in v24.1.0-beta.2 that could occur when planning statistics collection on a table with a [virtual computed column]({% link {{ page.version.version }}/computed-columns.md %}#virtual-computed-columns) using a user-defined type when the newly-introduced [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.stats.virtual_computed_columns.enabled` is set to `true`. (The setting was introduced in v24.1.0-alpha.1, and defaults to `true`.) [#124060][#124060]
+- Previously, [`ALTER TABLE ... ALTER PRIMARY KEY`]({% link {{ page.version.version }}/alter-table.md %}#alter-primary-key) could hang if the table had any indexes which were referred to by [views]({% link {{ page.version.version }}/views.md %}) or [functions]({% link {{ page.version.version }}/functions-and-operators.md %}) using the force index syntax. This is now fixed. [#124323][#124323]
+- Fixed a bug where CockroachDB would ignore the `max_decimal_digits` argument of the `st_geojson` [builtin function]({% link {{ page.version.version }}/functions-and-operators.md %}#built-in-functions) and would use the default instead. The bug was only present in v24.1.0. [#124502][#124502]
+- Scattering a range with a replication factor of 1 now no longer erroneously up-replicates the range to two replicas. Leases will also no longer thrash between nodes when perturbed with a replication factor of 1. [#124453][#124453]
```

Comment:
```suggestion
- Fixed bugs where scattering a range with a replication factor of 1 could erroneously up-replicate the range to two replicas, and where a lease could thrash between nodes when perturbed with a replication factor of 1. [#124453][#124453]
```
I'm not really sure what the last sentence means, so maybe this is an incorrect edit.